README.md

```diff
@@ -1,2 +1,56 @@
 # gl_rubocop
+
 A shareable configuration of rules we use at Give Lively to lint our Ruby code.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'gl_rubocop', require: false
+```
+
+And then execute:
+
+```bash
+ bundle install
+```
+
+Or install it yourself as:
+
+```bash
+ gem install gl_lint
+```
+
+And finally add this to the top of your project's RuboCop configuration file:
+
+```yml
+ inherit_gem:
+    rubocop-shopify: rubocop.yml
```

Comment:
I don't think you want `rubocop-shopify` here

Reply:
Nope, that was an oversight from me while using other READMEs as examples to make this one from